### PR TITLE
numpy version update for openvino compatibility

### PR DESCRIPTION
**Pull request by dongkwan-kim01**
### Changes

Numpy version update from `1.22.1` to `1.23.1`

### Reason for changes

Error in the openvino test in the CI server
image

### Tests

Testing with PR in CI

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def find_version(*file_paths):
     "docs": [],
     "tf": [
         "tensorflow~=2.8.2",
-        "numpy~=1.22.1"
+        "numpy~=1.23.1"
```
Comment (alexsu52) [inline setup.py:118]: Could you remove the dependency on the **exact** version of numpy from NNCF? I suggest to include numpy as common dependency of NNCF without version specification.

cc' @vshampor, @andrey-churkin, @MaximProshin, @AlexKoff88

Comment (dongkwan-kim01) [reply setup.py]: Good point, I'll move it to common dependency with range and test it on the CI and local server.

**Comment by dongkwan-kim01** [comment]
jenkins retest this please

**Comment by alexsu52** [comment]
retest this please

**Review by alexsu52 — APPROVED** [review]

**Pull request merged**